android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -758,8 +758,9 @@ public void startActivityForPluginWithResult(PluginCall call, Intent intent, int
     }
 
     /**
-     * Handle a request permission result by finding the that requested
-     * the permission and calling their permission handler
+     * Check for legacy Capacitor or Cordova plugins that may have registered to handle a permission
```

Comment:
@dwieeb How is this verbiage? See javadoc added in BridgeActivity line 167 also

Reply:
Perfect 👌